lib/Sema/InstrumenterSupport.h

```diff
@@ -0,0 +1,151 @@
+//===--- InstrumenterSupport.h - Instrumenter Support ---------------------===//
```

Comment:
Nitpick: please add the `-*- c++ -*-` header here, for the ~poor souls~ nice folks using emacs.